sklearn/metrics/pairwise.py

```diff
@@ -576,12 +578,23 @@ def _euclidean_distances_upcast(X, XX=None, Y=None, YY=None, batch_size=None):
     return distances
 
 
+# start is specified in the signature of `_argmin_min_reduce`
+# and of `_argmin_reduce` but is not used.
+# This is because the higher order `pairwise_distances_chunked`
+# function needs reduction functions that are passed as argument
+# to have a two arguments signature.
```

Comment:
I prefer to have this comment be in the body of `_argmin_min_reduce` and `_argmin_reduce`? There is a change the comment will get loss if `_argmin_reduce` gets moved around.

Reply:
Fixed with https://github.com/scikit-learn/scikit-learn/pull/22134/commits/ca6f379182e96f024193390b213cc4c90fc56598.